src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -88,6 +87,16 @@ public void testHtapTelemetry() throws SQLException {
     statement.execute("select 1");
   }
 
+  @Test
+  public void testDisableQueryContextCache() throws SQLException {
```

Comment:
What are we testing here? Are we testing for regression because of this new parameter? Because this test cannot tell by itself that the cache was actually disabled or not right?

Reply:
This is to test, if cache is disable still it's working without any error. Mainly because qcc was not initialized need to make sure no NPE occurs.

Reply:
Added check for query context.